utils/config.env

```diff
@@ -41,10 +41,10 @@ KAFKA_BRANCH=2.4
 #   REPOSITORY=<account>.dkr.ecr.us-west-2.amazonaws.com/confluentinc   # confluentinc repository at the specified ECR registry, images must be pulled separately
 #####################################################
 
-CP_VERSION_MAJOR=5.5
-CP_VERSION_FULL=5.5.0
-GIT_BRANCH=master
-JAR_VERSION=5.5.0-SNAPSHOT
+CP_VERSION_MAJOR="$CONFLUENT_MAJOR.$CONFLUENT_MINOR"
```

Comment:
Why is `CP_VERSION_MAJOR` necessary if there is already `CONFLUENT_SHORT`?

If `CP_VERSION_MAJOR` is used elsewhere in the repo, it should be replaced with `CONFLUENT_SHORT`

Reply:
Same as below, in an effort to keep the changes minimal in this PR, I just wanted to make the demos work w/ the values presented to us and this is the minimal change I could think of.  If we want to remove redundant names, I suggest we remove them in another PR to keep changes small.

Reply:
I addressed to use the SHORT version of the provided value, nice catch.  thanks